### ISSUE TITLE
Initially created Angular v12 project with angular-eslint does not detect errors when using 'ng lint'

### ISSUE THREAD

**Issue by Taras-R** [angular-eslint/angular-eslint#517]
By following the instructions i created new Angular v12 project and added angular-eslint with **ng add @angular-eslint/schematics**

_package.json_
![image](https://user-images.githubusercontent.com/8331527/120018929-78857300-bff0-11eb-8e34-ccd8ccee07f1.png)

_angular.json_
![image](https://user-images.githubusercontent.com/8331527/120018996-9226ba80-bff0-11eb-97df-fc9f57fcb141.png)

_.eslintrc.json_
`{
  "root": true,
  "ignorePatterns": [
    "projects/**/*"
  ],
  "overrides": [
    {
      "files": [
        "*.ts"
      ],
      "parserOptions": {
        "project": [
          "tsconfig.json",
          "e2e/tsconfig.json"
        ],
        "createDefaultProgram": true
      },
      "extends": [
        "plugin:@angular-eslint/recommended",
        "plugin:@angular-eslint/template/process-inline-templates"
      ],
      "rules": {
        "@angular-eslint/directive-selector": [
          "error",
          {
            "type": "attribute",
            "prefix": "app",
            "style": "camelCase"
          }
        ],
        "@angular-eslint/component-selector": [
          "error",
          {
            "type": "element",
            "prefix": "app",
            "style": "kebab-case"
          }
        ]
      }
    },
    {
      "files": [
        "*.html"
      ],
      "extends": [
        "plugin:@angular-eslint/template/recommended"
      ],
      "rules": {}
    }
  ]
}
`

It is the first time our team uses eslint in project, as before we used tslint Angular shipped with.

Now when we try to test some ugly code by **ng lint** command, it says just that all files passed linting and shows no errors. Nothing was manually modified in any settings of eslint. Please help, as we can not understand what we do wrong. Does it require some manual configuration of .eslintrc.json file? By the way installed VSCode eslint extension also does not highlight any error, but is enabled for the workspace and successfully runs.

I know it can look as dumb question, and maybe we just miss something very obvious, but we really use it for first time and never faced with such issues when used tslint in the past.

_code example_
![image](https://user-images.githubusercontent.com/8331527/120019374-0b261200-bff1-11eb-941d-b15b443144f8.png)

_console output_
![image](https://user-images.githubusercontent.com/8331527/120019439-1f6a0f00-bff1-11eb-9830-27bc5aeee0ad.png)

Thank you in advance!

**Comment by 9034725985** [comment]
I can reproduce this issue. 

![image](https://user-images.githubusercontent.com/1676445/120087797-46cfe300-c0a8-11eb-9e66-df00764f01b6.png)
![image](https://user-images.githubusercontent.com/1676445/120087814-77178180-c0a8-11eb-9753-9c9c0619856a.png)

**Comment by timdeschryver** [comment]
If I understand this issue correctly, you would have expected that the highlighted code would cause lint errors?
If that's the case you can add `plugin:@typescript-eslint/recommended` to the ESLint configuration.

**Comment by Taras-R** [comment]
@timdeschryver @9034725985 Thank you guys for your attention, it was very helpful!
For others angular-eslint first-time users facing the same issue, i've aded following to **.eslintrc.json**
```
"extends": [
  "eslint:recommended",
  "plugin:@typescript-eslint/recommended",
  "plugin:@angular-eslint/recommended",
  "plugin:@angular-eslint/template/process-inline-templates"
],
```
Now it is highlighting and properly displayed in console when using **ng lint**
Issue is closed.

**Comment by 9034725985** [comment]
Thank you, @Taras-R @timdeschryver  
![image](https://user-images.githubusercontent.com/1676445/120101564-d951a000-c103-11eb-9e6b-9f55b677c1ce.png)
